server/routes/forms.js

```diff
@@ -21,6 +21,19 @@ module.exports = ({ formService }) => router => {
         licenceStatus,
       } = res.locals
 
+      const abuseAndBehaviours = getIn(res.locals.licence, [
+        'licence',
+        'licenceConditions',
+        'additional',
+        'COMPLYREQUIREMENTS',
+        'abuseAndBehaviours',
+      ])
+
+      if (Array.isArray(abuseAndBehaviours)) {
+        licence.licenceConditions.additional.COMPLYREQUIREMENTS.abuseAndBehaviours = abuseAndBehaviours.map(item => {
+          return ` ${item}`
+        })
+      }
```

Comment:
I don't think this is the right place to format a specific data item for printing.
Line 40 below, calls the confusingly named getCurfewAddressCheckData function.  It is badly named because it does a *lot* more than retrieve curfew address check data.  This function formats all the licence data for the address check form.
Formatting the licence conditions is delegated to conditionsService.js (getFullTextForApprovedConditions).
That is where this piece of formatting should go.  Probably in the function 'getConditionText', which looks surprisingly like the other small formatting function used by the licence PDF formatting code.  A similar tweak to this function should do the trick.  If it turns out that the changes are identical to those on the other function then I'd suggest extracting that code in to a separate module to remove the duplication.

Reply:
Thanks. I'll have another go

Reply:
The logic above (lines 24 to 36) was included to insert a space between the Behaviours. Have removed it and changed populateLicenceWithConditions() inside server/services/conditionsService. getFullTextForApprovedConditions  so that it places a space before each condition